src/data/model/DataCase.java

```diff
@@ -129,4 +139,44 @@ public void setInquiry(String inquiry) {
     this.inquiry = inquiry;
   }
 
+  /**
+   * Get informed about rights status
+   *
+   * @return informedAboutRights
+   */
+  @Override
+  public boolean isInformedAboutRights() {
+    return this.informedAboutRights;
```

Comment:
No reason to ```this.``` , the same goes for the other getters and setters. (But not wrong)

Reply:
I like-a da this